fvm/bootstrap.go

```diff
@@ -380,6 +381,9 @@ func (b *bootstrapExecutor) Execute() error {
 	// set the list of nodes which are allowed to stake in this network
 	b.setStakingAllowlist(service, b.identities.NodeIDs())
 
+	// sets up the EVM environment
+	b.setupEVM(service)
```

Comment:
should this be executed if fvm enabled is false?

Reply:
for bootstrapping, we don't usually do it conditionally on the code, I can change this approach but would be the first time we are doing the bootstrapping that way.

Reply:
@janezpodhostnik thoughts ?

Reply:
We do some bootstrap things conditionally on the code,  like setup the fees or the initial funds, some epoch things....

On one hand skipping this when we don't need it (most likely in tests) would speed up the tests, but on the other, it would be better to have the tests be the closest as possible to what we then later run. 

I personally would preferrer this be guarded by an option so we can have faster tests when we are testing unrelated things.